### ISSUE TITLE
Failed to load API definition

### ISSUE THREAD

**Issue by erossini** [yuka1984/azure-functions-extensions-swashbuckle#45]
I created a simple project where I added the latest version of the extension. I get this error:

![image](https://user-images.githubusercontent.com/9497415/81652137-2b595200-942b-11ea-9d45-491f9a7903ad.png)

> Failed to load API definition
> Fetch error 
> Unauthorized

I put my code [here](https://github.com/erossini/AzureFunctionsSwashbuckle). I followed the steps in the project page. It was working perfectly on Azure until a couple of day ago. Locally is working.

```
{
  "schemaValidationMessages": [
    {
      "level": "error",
      "message": "Can't read from file https://azuks-test-q001.azurewebsites.net/api/swagger/json"
    }
  ]
}
```

## Update
For same reason, the `code` param can't be the same for `json` and `UI`. If you open your Swagger url with **Get Function url** and copy the url from UI function, it is working.

![image](https://user-images.githubusercontent.com/9497415/81725857-5faa2e00-947e-11ea-8da8-48314dc517ae.png)

**Comment by vitalybibikov** [comment]
Hi @erossini,

```
    <PackageReference Include="AzureExtensions.Swashbuckle" Version="3.1.6" />
    <PackageReference Include="AzureFunctions.Extensions.Swashbuckle" Version="1.4.4" />
```

1.4.4 is an old version, please use 3.1.6 only as it is a fork with updates

**Comment by erossini** [comment]
In the Nuget Package Manager I don't have this version

![image](https://user-images.githubusercontent.com/9497415/82664126-f28a5b80-9c28-11ea-8b53-348d47960f06.png)

**Comment by erossini** [comment]
You meant you have to use this one?

![image](https://user-images.githubusercontent.com/9497415/82664247-32514300-9c29-11ea-9642-2cff019552c3.png)

**Comment by vitalybibikov** [comment]
Yep, exactly. I've forked it, due to the repo is currently stale.

Fork => https://github.com/yuka1984/azure-functions-extensions-swashbuckle
Nuget
Nuget => https://www.nuget.org/packages/AzureExtensions.Swashbuckle/

**Comment by lprit** [comment]
@erossini Did upgrading it fix it for you? I'm on 3.1.6 and my swagger/ui function still can't access my swagger/json function without setting the swagger/json auth level to anonymous. Same error.

**Comment by erossini** [comment]
> @erossini Did upgrading it fix it for you? I'm on 3.1.6 and my swagger/ui function still can't access my swagger/json function without setting the swagger/json auth level to anonymous. Same error.

Yes, but nothing changed, same errors. You can open the json copying the url of `swagger` and paste it in the location bar. This is working

**Comment by erossini** [comment]
Also, I noticed that there is no description in the functions like the xml documentation although I added in the `host.json` 

    "extensions": {
      "Swashbuckle": {
        "XmlPath": "Payments.xml"
      }
    }

![image](https://user-images.githubusercontent.com/9497415/82882110-89f8f280-9f38-11ea-950b-54cd069ef263.png)
